Src/WitsmlExplorer.Desktop/src/preload/preload.ts

```diff
@@ -1,5 +1,10 @@
 import { contextBridge, ipcRenderer } from "electron";
 
 contextBridge.exposeInMainWorld("electronAPI", {
-  getConfig: () => ipcRenderer.invoke("getConfig")
+  getConfig: () => ipcRenderer.invoke("getConfig"),
+  onCloseWindow: (callback: any) =>
+    ipcRenderer.on("closeWindow", () => callback()),
+  removeCloseWindow: () => ipcRenderer.removeAllListeners("closeWindow"),
```

Comment:
Maybe rename to removeCloseWindowListener?

Reply:
Fixed